### PR TITLE
[Add] project_budget

### PR DESCRIPTION
**Pull request by patrickrwilson**
Adding project_budget which integrates account_budget_oca with projects so you can manage your budget directly on the project.

### CODE REVIEW

**Comment by patrickrwilson** [comment]
ping @max3903

**Comment by patrickrwilson** [comment]
@max3903 do you mind requesting some reviewers and setting tags on this pr?

**Review by bodedra — APPROVED** [review]
@patrickrwilson Nice work.

Functional review on runbot. It works great. :+1:

**Review by nikul-serpentcs — APPROVED** [review]
@patrickrwilson Code Review LGTM :+1: 

Only Create a README base on this  https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

File: project_budget/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    'summary': "Adds budget management to projects.",
+    'author': "Patrick Wilson,Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/project',
+    'category': 'Project Management',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'depends': [
+        'project',
+        'account_budget_oca',
+        'hr_timesheet',
```
Comment (rafaelbn) [inline project_budget/__manifest__.py:14]: Why this dependency?

File: project_budget/README.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+==============
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Have user rights for Project Budget viewer or manager group.
+#. Open a project and view the Budget tab.
+#. Create a new budget. The budget lines are visible on the project.
```
Comment (rafaelbn) [inline project_budget/README.rst:12]: I follower this steps and I get this (screenshot), but after that I don't know how to test it. I put some timesheet to the project but nothing happens.

**Review by rafaelbn — COMMENTED** [review]
I'm unable to test this module as complex to use it. I have gone first to the readme of account_budget_oca but despite trying step by step I don't know how to spent the budget

I wish you take care in the change from v11 to v12, now 1 analytic account could have multiple proyect associated. So you can make a budget for a project and use it in several project when all are associated yo the same analytic account.

File: project_budget/README.rst (hunk 2 above)
Comment (patrickrwilson) [reply project_budget/README.rst:12]: You've pretty much got it. The budget itself is managed in the oca budget module and this just makes the budget visible in the project but if you want to test further, you can create some accounting transactions and link the analytic account it should show the practical amounts.

File: project_budget/__manifest__.py (hunk 1 above)
Comment (patrickrwilson) [reply project_budget/__manifest__.py:14]: project analytic accounting is part of the hr_timesheet which is needed.

**Comment by patrickrwilson** [comment]
Thanks again for reviewing, did I miss anything or is this ready to be merged?

**Review by osi-scampbell — APPROVED** [review]

**Comment by max3903** [comment]
@sbidoul Can you help us here? @patrickrwilson is listed as the maintainer but he still cannot merge.

**Comment by sbidoul** [comment]
Adding new maintainers to an add-on is a PSC prerogative. That holds true
for new addons too. I think it's such a case.

**Comment by max3903** [comment]
@sbidoul Thanks!

**Pull request merged**